src/test/java/ua/orlov/springcoregym/controller/integration/trainee/TraineeControllerIntegrationTest.java

```diff
@@ -0,0 +1,648 @@
+package ua.orlov.springcoregym.controller.integration.trainee;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPatch;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import ua.orlov.springcoregym.controller.integration.config.LoginComponent;
+import ua.orlov.springcoregym.controller.integration.config.HttpRequest;
+import ua.orlov.springcoregym.dto.trainee.*;
+import ua.orlov.springcoregym.dto.user.UsernameIsActiveUser;
+import ua.orlov.springcoregym.dto.user.UsernameUser;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@Sql(scripts = "/sql/trainee/populate_encrypted_trainee.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_CLASS)
+@Sql(scripts = "/sql/prune_tables.sql", executionPhase = Sql.ExecutionPhase.AFTER_TEST_CLASS)
+public class TraineeControllerIntegrationTest {
```

Comment:
Let's use a pattern for intergration test naming ... IT, words IntegrationTest should not be used in the integration tests naming. Also it should be done some configuration for in maven plugins